pre_commit_hooks_dependency_sync/main.py

```
@@ -61,7 +61,7 @@ def get_installed_packages(lockfile: Path, package_manager: str) -> dict[str, st
     raise NotImplementedError(f"Package manager {package_manager!r} not implemented")
 
 
-def main() -> None:
+def main() -> None:  # noqa: PLR0912
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0912`)

Remove unused `noqa` directive

(RUF100)